components/carousel/carousel.css

```diff
@@ -0,0 +1,83 @@
+.ui-carousel {
+  overflow: hidden;
+  position: relative;
+}
+
+.ui-carousel-item {
```

Comment:
I think most styles are using the BEM style of defining blocks, basically, blocks like this should be defined in this style `.ui-carousel__item`, `.ui-carousel__item-item`, `.ui-carousel__markers`, etc.